app.js

```diff
@@ -133,8 +133,9 @@ const intToRoman = function(num) {
 
 const integerInput = document.querySelector('#integer');
 integerInput.addEventListener('change', e => {
-  const integer = e.target.value;
+  const integer = Math.trunc(e.target.value);
   const romanNumeral = intToRoman(integer);
   const romanNumeralDisplay = document.querySelector('#roman-numeral');
+  integerInput.value = integer;
```

Comment:
Not sure what this line is doing .. ? I commented it out and didn't notice any difference

Reply:
If the user enters a decimal, it will be truncated and then the input will be set to the truncated integer. If you comment it out and enter an integer, there won't be any difference. If you comment it out and enter a decimal, that is the only time there will be a difference.